lib/devices/sensor.js

```diff
@@ -217,6 +227,11 @@ module.exports = function(device, api, capabilities) {
           .getCharacteristic(Characteristic.CurrentRelativeHumidity)
           .updateValue(capValue('measure_humidity'));
         break;
+      case 'measure_pm25': // AirQualitySensor
+        homekitAccessory
+          .getService(Service.AirQualitySensor)
+          .getCharacteristic(Characteristic.PM2_5Density)
+          .updateValue(value);
```

Comment:
This should probably use `capValue(value)` instead of just `value` (from what I remember, the reason for this is that Homeykit only exposes the first "instance" of a capability and `capValue` needs to select the value of that first capability).